cookiecutter.json

```diff
@@ -89,7 +89,7 @@
       "nl": "Nederlands",
       "fi": "Suomi"
     },
-    "github_organization": "GitHub Organization",
+    "github_organization": "GitHub User or Organization",
```

Comment:
GitHub calls it `Username`.

<img width="492" alt="Screenshot 2023-10-03 at 9 58 22 AM" src="https://github.com/collective/cookiecutter-plone-starter/assets/102112/f3a15b3f-4eeb-4f6e-8783-8f079e90d1e8">